build_tools/kickstarts/rhel-7-cdk-vagrant.ks

```diff
@@ -58,6 +84,9 @@ openshift2nulecule
 # Workaround for BZ1336857
 restorecon -v /usr/bin/docker*
 
+LANG="en_US"
+echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
```

Comment:
As per doc http://rpm.org/api/4.4.2.2/config_macros.html, since we only care about en_US then package installed should have only en_US locale instead of other languages.

#	A colon separated list of desired locales to be installed;
#	"all" means install all locale specific files.
#	
%_install_langs	all